src/main/webapp/index.html

```diff
@@ -13,25 +13,26 @@ <h1 class="name">Events</h1>
     </div>
 
     <div id="add-button">
-        <button class="create-event" onclick="openEventPopup()">+</button>
+        <button class="create-event" onclick="window.openEventPopup()">+</button>
```

Comment:
Can this be popup.openEventPopup

This will make it clear to the user where it originates.

Reply:
As I discussed with Maksim earlier, I am not allowed to use functions outside of the module (in HTML). The only solution we found was to use the global object window to make the functions visible in the HTML page